### PR TITLE
Document translations on Weblate

### PR DESCRIPTION
**Pull request by Findus23**
To be updated during the migration

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: docs/maintaining-translations.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+```json
+{
+    "squash": "language",
+    "append_trailers": true,
+    "commit_message": ""
+}
+```
+
+Removes translation keys no longer in `en.json`
```
Comment (sgiehl) [inline docs/maintaining-translations.md]: Guess this needs to be moved some lines up

Comment (Findus23) [reply docs/maintaining-translations.md]: Indeed

**Comment by sgiehl** [comment]
@tsteur could you maybe have a rough look through this translation documentation and check if you think anything would be missing? We have tried to move most stuff from our internal docs here, as there shouldn't be a reason for not having it documented public.

File: docs/maintaining-translations.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+### List of involved translators
+
+On https://matomo.org/translations we are showing which languages Matomo is available in, as well as a list of translators for each language. This list is actually created through a Matomo API request (`LanguagesManager.getAvailableLanguagesInfo`). Therefore, the list of translators is stored within the base translations files (`General_TranslatorName`).
+
+Note: As the list is maintained in the translation files, the list on matomo.org will only be updated, once the translations have been update on GitHub, a new version was released and deployed.
+
+
+## Update data from Unicode CLDR
+
+The translations of the *Intl* plugin are automatically converted from the Unicode Common Locale Data Repository. To update them, check https://github.com/unicode-org/cldr-json/releases for the latest stable release of CLDR and update `$CLDRVersion` in ` plugins/Intl/Commands/GenerateIntl.php`. Afterwards you can run `php console translations:generate-intl-data` and commit the changed files in `plugins/Intl/lang`.
```
Comment (tsteur) [inline docs/maintaining-translations.md:185]: @sgiehl this one we can remove as we have an internal process document for it

**Review by tsteur — COMMENTED** [review]
Sounds good @sgiehl we still also need to update internal docs, and also internally add information why we left transifex, why we chose weblate, what our deal is with them / our limits etc.

File: docs/maintaining-translations.md (hunk 2 above)
Comment (sgiehl) [reply docs/maintaining-translations.md:185]: Won't hurt to have it here. Also this is required to run when adding a new language, as the Intl data is required in this case. See https://github.com/matomo-org/developer-documentation/pull/523/files#diff-3e2d80ac2142428f04b2ca2e431263b92db4ea347f9e2f56e3451fc50d810d4fR96

**Pull request merged**